### PR TITLE
i18n(es): update `server side rendering` page

### PR DESCRIPTION
**Pull request by antonyfaris**
#### What kind of changes does this PR include?

- Translated content
- New or updated content

#### Description

-  Update with [these changes](https://github.com/withastro/docs/commits/main/src/pages/en/guides/server-side-rendering.md?since=2022-08-22T23:11:59.000Z)

### CODE REVIEW

File: src/pages/es/guides/server-side-rendering.md [hunk 1]
```diff
@@ -200,3 +211,34 @@ export async function post({ request }) {
   return new Response(JSON.stringify(responseData), { status: 200 });
 }
 ```
+
+
+### Redirects
+
+Ya que `Astro.redirect` no está disponible rutas API puedes usar [`Response.redirect`](https://developer.mozilla.org/en-US/docs/Web/API/Response/redirect).
```
Comment (fredoist) [inline src/pages/es/guides/server-side-rendering.md:218]:
```suggestion
Ya que `Astro.redirect` no está disponible en las rutas de API puedes usar [`Response.redirect`](https://developer.mozilla.org/en-US/docs/Web/API/Response/redirect).
```

File: src/pages/es/guides/server-side-rendering.md [hunk 2]
```diff
@@ -200,3 +211,34 @@ export async function post({ request }) {
   return new Response(JSON.stringify(responseData), { status: 200 });
 }
 ```
+
+
+### Redirects
```
Comment (fredoist) [inline src/pages/es/guides/server-side-rendering.md:216]: Should we translate this title or keep it? If so also check for broken links.

```suggestion
### Redirecciones
```

File: src/pages/es/guides/server-side-rendering.md [hunk 3]
```diff
@@ -24,24 +24,29 @@ Los siguientes adaptadores están disponibles hoy y habrán muchos más en el fu
 - [Node.js](/es/guides/integrations-guide/node/)
 - [Vercel](/es/guides/integrations-guide/vercel/)
 
-Puedes encontrar las instrucciones de configuración en los enlaces anteriores de acuerdo al adaptador. Para completar los siguientes dos pasos (usaremos `my-adapter` como ejemplo) para habilitar SSR.
+Puedes añadir cualquiera de los adaptadores oficiales con el siguiente comando `astro add`. Esto instalará el adaptador y hará los cambios apropiados a tu archivo `astro.config.mjs` en un solo paso. Por ejemplo, para instalar el adaptador de Netlify, ejecuta:
```
Comment (agustinmulet) [inline src/pages/es/guides/server-side-rendering.md:27]:
```suggestion
Puedes añadir cualquiera de los adaptadores oficiales con el comando `astro add`. Esto instalará el adaptador y hará los cambios apropiados a tu archivo `astro.config.mjs` en un solo paso. Por ejemplo, para instalar el adaptador de Netlify, ejecuta:
```

File: src/pages/es/guides/server-side-rendering.md [hunk 4]
```diff
@@ -63,6 +68,12 @@ const cookie = Astro.request.headers.get('cookie');
 </html>
 ```
 
+:::caution
+Las características listadas a continuación solo están disponibles a nivel de página. (No las puedes usar dentro de componentes, incluyendo componentes de *layout*.)
+
+Esto se debe a que estás características [modifican las respuestas de los *headers*](https://developer.mozilla.org/en-US/docs/Glossary/Response_header), los cuales no pueden ser modificados después de ser enviados al navegador. In SSR mode, Astro usa *HTML streaming* para enviar cada componente al navegador mientras los renderiza. Esto asegura que el usuario vea tu HTML lo más rápido posible, pero también significa que para el momento que Astro ejecuta el código de tu componente, ya se ha enviado los *Response headers*.
```
Comment (fredoist) [inline src/pages/es/guides/server-side-rendering.md:74]:
```suggestion
Esto se debe a que estás características [modifican las respuestas de los *headers*](https://developer.mozilla.org/en-US/docs/Glossary/Response_header), los cuales no pueden ser modificados después de ser enviados al navegador. En modo SSR, Astro usa *HTML streaming* para enviar cada componente al navegador mientras los renderiza. Esto asegura que el usuario vea tu HTML lo más rápido posible, pero también significa que para el momento que Astro ejecuta el código de tu componente, ya se ha enviado los *Response headers*.
```

Comment (agustinmulet) [inline src/pages/es/guides/server-side-rendering.md:74]:
```suggestion
Esto se debe a que estas características [modifican las respuestas de los *headers*](https://developer.mozilla.org/es/docs/Glossary/Response_header), los cuales no pueden ser modificados después de ser enviados al navegador. En el modo SSR, Astro usa *HTML streaming* para enviar cada componente al navegador mientras los renderiza. Esto asegura que el usuario vea tu HTML lo más rápido posible, pero también significa que para el momento que Astro ejecuta el código de tu componente, ya se han enviado los *Response headers*.
```

**Review by fredoist — CHANGES_REQUESTED** [review]
Suggested some changes.

File: src/pages/es/guides/server-side-rendering.md (hunk 2 above)
Comment (agustinmulet) [inline src/pages/es/guides/server-side-rendering.md:216]:
```suggestion
### Redirecciones
```

File: src/pages/es/guides/server-side-rendering.md (hunk 1 above)
Comment (agustinmulet) [inline src/pages/es/guides/server-side-rendering.md:218]:
```suggestion
Ya que `Astro.redirect` no está disponible en las rutas API, puedes usar [`Response.redirect`](https://developer.mozilla.org/en-US/docs/Web/API/Response/redirect).
```

File: src/pages/es/guides/server-side-rendering.md [hunk 5]
```diff
@@ -200,3 +211,34 @@ export async function post({ request }) {
...
+      status: 404,
+      statusText: 'Not found'
+    });
+  }
+
+  return Response.redirect(link, 307);
+}
+```
+
+`Response.redirect` requiere que pases la URL completa. Para redirecciones locales, puedes usar `request.url` como base con [el constructor `URL`](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) para generar la URL absoluta:
```
Comment (agustinmulet) [inline src/pages/es/guides/server-side-rendering.md:238]:
```suggestion
`Response.redirect` requiere que pases la URL completa. Para redirecciones locales, puedes usar `request.url` como base con [el constructor `URL`](https://developer.mozilla.org/es/docs/Web/API/URL/URL) para generar la URL absoluta:
```

**Review by agustinmulet — COMMENTED** [review]
Reviewed! Fantastic job! 💪

**Pull request merged**